components/resources/brave_components_strings.grd

```diff
@@ -854,6 +854,12 @@
       <message name="IDS_BRAVE_UI_SHOW_ALL" desc="">Show All</message>
       <message name="IDS_BRAVE_UI_VIEWED_SITES" desc="">Sites Viewed</message>
 
+      <message name="IDS_BRAVE_UI_PROMO_LEARN_MORE" desc="">Learn more</message>
+      <message name="IDS_BRAVE_UI_PROMO_DISMISS" desc="">Don't show again</message>
+
+      <message name="IDS_BRAVE_UI_BITFLYER_VERIFICATION_PROMO_TITLE" desc="">Verify your wallet</message>
```

Comment:
Do we want to change these strings to get rid of the "wallet" language?

Reply:
Great point!

@Miyayes I'll just drop the `your wallet` portion of the string on Android, if that makes sense.